doc/man/8/rbd.rst

```diff
@@ -647,6 +647,8 @@ Per client instance `rbd device map` options:
   since 4.2 this can be used to ensure that `rbd device unmap` eventually
   times out when there is no network connection to a cluster.
 
+* lock_timeout=x - A timeout on waiting exclusive lock.
```

Comment:
Change this to
```
* lock_timeout=x - A timeout on waiting for the acquisition of exclusive lock
  (since 4.17, default is 0 seconds, meaning no timeout).
```